src/scripts/get-shipyard-environment.sh

```diff
@@ -6,6 +6,11 @@ ORG="${CIRCLE_PROJECT_USERNAME}"
 REPO="${CIRCLE_PROJECT_REPONAME}"
 BRANCH="${CIRCLE_BRANCH}"
 
+# Get auth token
+export SHIPYARD_API_TOKEN=${!PARAM_SHIPYARD_TOKEN}
+[ -z "$SHIPYARD_API_TOKEN" ] && echo "A Shipyard API token must be supplied. Check the \"api-token\" parameter." && exit 1
+echo "export SHIPYARD_API_TOKEN=\"${SHIPYARD_API_TOKEN}\"" >> "$BASH_ENV"
```

Comment:
We don't need to add this to the Circle environment - the user already has it, they provide it themselves.